### ISSUE TITLE
🐛 [BUG]: Nullsafety.0 Received status code 502 from server: Bad Gateway

### ISSUE THREAD

**Issue by mahdi-malv** [pusheco/pushe-flutter#22]
Log:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':app:releaseUnitTestRuntimeClasspath'.
> Could not resolve com.huawei.hms:push:5.0.2.300.
Required by:
project :app > project :pushe_flutter > co.pushe.plus:hms:2.5.0
> Could not resolve com.huawei.hms:push:5.0.2.300.
> Could not get resource 'https://dl.bintray.com/pushe/preview/com/huawei/hms/push/5.0.2.300/push-5.0.2.300.pom'.
> Could not HEAD 'https://dl.bintray.com/pushe/preview/com/huawei/hms/push/5.0.2.300/push-5.0.2.300.pom'. Received status code 502 from server: Bad Gateway
> Could not resolve com.huawei.hms:ads-identifier:3.4.29.301.
Required by:
project :app > project :pushe_flutter > co.pushe.plus:hms:2.5.0
> Could not resolve com.huawei.hms:ads-identifier:3.4.29.301.
> Could not get resource 'https://dl.bintray.com/pushe/preview/com/huawei/hms/ads-identifier/3.4.29.301/ads-identifier-3.4.29.301.pom'.
> Could not HEAD 'https://dl.bintray.com/pushe/preview/com/huawei/hms/ads-identifier/3.4.29.301/ads-identifier-3.4.29.301.pom'. Received status code 502 from server: Bad Gateway

```

**Comment by mahdi-malv** [comment]
Reason: 
Remote url "`https://dl.bintray.com/pushe/preview`" is deprecated and now results to 502 error.
This remote isn't used

**Comment by mahdi-malv** [comment]
I've published pushe_flutter:v[2.5.1-nullsafety.1](https://pub.dev/packages/pushe_flutter/versions/2.5.1-nullsafety.1) which removes this URL and therefore, packages will be resolved to `mavenCentral`
